lib/menu.rb

```diff
@@ -0,0 +1,33 @@
+class Menu
+
+  require 'ap'
```

Comment:
your requires should be at the top of the file really, not inside the class definition